CHANGELOG.md

```diff
@@ -10,6 +10,7 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 
 ### Added
 
+- An HTTP middleware in addition to the existing HTTP handler. The middleware is constructed once and afterwards can take different handlers as arguments, whereas the handler wraps a single http.Handler during construction. This allows better compatiblity with packages like github.com/urfave/negroni and github.com/go-chi/chi.
```

Comment:
Also we should add that to the `### Removed`

```
- Remove `Handler` type in `go.opentelemetry.io/contrib/instrumentation/net/http/otelhttp`. (#2964)
```

Reply:
Done